File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    # your code here
    for x in range(len(array_to_search_through)):
        if(array_to_search_through[x]==value_to_find):
            return x
    return None

File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    const output = [];
    for(var i=0;i<arrayToSearchThrough.length;i++){
        if(arrayToSearchThrough[i]==valueToFind){
            output.push(i);
        }
    }
    return output;
};

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    for(var i=0;i<arrayToSearchThrough.length;i++){
        if(arrayToSearchThrough[i]==valueToFind){
            return i;
        }
    }
    return undefined;
};

